website/docs/concepts/html/elements.md

```diff
@@ -19,6 +19,43 @@ html! {
 }
 ```
 
+## Boolean Attributes 
+
+Some content attributes (e.g checked, hidden, required) are called boolean attributes. In Yew, 
+boolean attributes need to be explicitly set to 'true':
```

Comment:
This should be reworded since you don't need to set them to the literal 'true', it just has to be a `bool` value.